common/org.testeditor.dsl.common/META-INF/MANIFEST.MF

```diff
@@ -9,15 +9,15 @@ Bundle-Vendor: Signal Iduna Corporation
 Require-Bundle: org.eclipse.xtend.lib.macro,
  org.eclipse.xtext,
  org.testeditor.logging,
- org.eclipse.core.runtime,
+ org.eclipse.core.runtime;visibility:=reexport,
  org.eclipse.emf.ecore;visibility:=reexport,
  org.eclipse.xtext.xbase.lib,
  org.eclipse.emf.ecore.xcore.lib,
  org.eclipse.e4.core.contexts;bundle-version="1.5.0";visibility:=reexport,
  org.eclipse.ui.workbench;bundle-version="3.108.0",
  org.eclipse.core.resources;bundle-version="3.11.0",
  org.testeditor.dsl.common.model;bundle-version="1.2.0",
- org.eclipse.jdt.core;bundle-version="3.12.0",
+ org.eclipse.jdt.core;bundle-version="3.12.0";visibility:=reexport,
```

Comment:
If this dependency is not reexported I would have to add this dependency into packages like org.testeditor.dsl.tcl (using ClasspathUtil to derive packages for qualified names).